apps/drec-api/src/utils/get-commissioning-date-range.ts

```diff
@@ -0,0 +1,27 @@
+import { CommissioningDateRange } from './enums';
+
+export const getDateRangeFromYear = (
+  commissioningDate: string,
+): CommissioningDateRange => {
+  const year = new Date(commissioningDate).getFullYear();
+  const currentYear = new Date().getFullYear();
+  const range = currentYear - year;
+  if (range === 0) {
+    return CommissioningDateRange.Year_1;
+  } else if (range === 1) {
+    return CommissioningDateRange.Year_2;
+  } else if (range === 2) {
+    return CommissioningDateRange.Year_3;
+  } else if (range === 3) {
+    return CommissioningDateRange.Year_4;
+  } else if (range === 4) {
+    return CommissioningDateRange.Year_5;
+  } else if (range >= 6 && range <= 10) {
+    return CommissioningDateRange.Between_years_6_10;
+  } else if (range >= 11 && range <= 15) {
+    return CommissioningDateRange.Between_years_11_15;
+  } else if (range >= 15) {
+    return CommissioningDateRange.Above_15_years;
+  }
+  return CommissioningDateRange.Year_1;
```

Comment:
`switch ... case` clause would suit this use-case better.

Reply:
I thought about it but when doing comparison it won`t work propertly and it will be slower.

Reply:
Same issue here when comparing if a value is between x and y.

Reply:
By the way, this return is "whatever" (why 1 Year, if this code is unreachable technically), I would drop it at all since it's not necessary if you change last statement to `else` or drop it at all at treat this as "any other value to be returned" outside `if`. Or this should throw with "unreachable reached, check the data:" whatever :p